src/System.Runtime.Serialization.Xml/tests/ReflectionOnly/System.Runtime.Serialization.Xml.ReflectionOnly.Tests.csproj

```diff
@@ -9,13 +9,17 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Release|AnyCPU'" />
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp'">
+    <Compile Include="$(TestSourceFolder)..\SerializationTypes.netcoreapp.cs" />
+    <Compile Include="$(TestSourceFolder)..\DataContractSerializer.netcoreapp.cs" />
+    <Compile Include="$(TestSourceFolder)..\DataContractSerializerStressTests.cs" />
```

Comment:
Well, I didn't add the sufix to that file because was the only file with that name and all of its contents where already netcoreapp specific, but I could add it so that it is easier to distinguish in the future.